scripts/dbsetup.sql

```diff
@@ -68,3 +76,10 @@ CREATE TRIGGER set_timestamp_healthit_products
 BEFORE UPDATE ON healthit_products
 FOR EACH ROW
 EXECUTE PROCEDURE trigger_set_timestamp();
+
+
+CREATE or REPLACE VIEW org_mapping AS
+SELECT endpts.url, orgs.name, orgs.Location->>'state'
+FROM endpoint_organization AS links
+LEFT JOIN fhir_endpoints AS endpts ON links.endpoint_id = endpts.id
+LEFT JOIN npi_organizations AS orgs ON links.organization_id = orgs.id;
```

Comment:
@mrnosal can you comment on if it would be helpful for more info to be included in this join table?
this table currently let's you easily see together:
endpoint url, organization name, organization state.

i'm thinking adding endpoint vendor to this list would be good.
we also discussed going down to the zipcode level.